kansa/lib/public.js

```diff
@@ -70,9 +86,10 @@ function lookupPerson(req, res, next) {
   }
   req.app.locals.db.any(`
     SELECT id, membership, preferred_name(p) AS name
-      FROM people p
-     WHERE ${queryParts.join(' AND ')}
-           AND membership NOT IN ('Child', 'KidInTow')`, queryValues
+    FROM people p
+      LEFT JOIN membership_types m USING (membership)
+    WHERE ${queryParts.join(' AND ')} AND
+      m.allow_lookup = true`, queryValues
```

Comment:
The `queryParts` construction logic is looking quite complex. Could we extract this to a function?

This would make it easy to write a little test ;-).